concordium-std/src/types.rs

```diff
@@ -645,6 +646,13 @@ pub struct AttributesCursor {
     pub(crate) remaining_items:  u16,
 }
 
+/// An iterator over the attributes of a policy.
+/// The iterator returns pairs of [`AttributeTag`] and [`AttributeValue`].
+pub struct PolicyAttributesIter<'a, P: HasPolicy> {
+    pub iter: &'a mut P,
+    pub buf:  [u8; 31],
```

Comment:
I don't think these fields needs to be public

Reply:
They shouldn't be. Especially the `buf`.